README.md

```diff
@@ -4,15 +4,15 @@ ScreenPy Selenium
 [![Build Status](../../actions/workflows/tests.yml/badge.svg)](../../actions/workflows/tests.yml)
 [![Build Status](../../actions/workflows/lint.yml/badge.svg)](../../actions/workflows/lint.yml)
 
-[![Supported Versions](https://img.shields.io/pypi/pyversions/screenpy-selenium.svg)](https://pypi.org/project/screenpy)
+[![Supported Versions](https://img.shields.io/pypi/pyversions/screenpy_selenium.svg)](https://pypi.org/project/screenpy_selenium)
 [![Checked with mypy](http://www.mypy-lang.org/static/mypy_badge.svg)](http://mypy-lang.org/)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![Imports: isort](https://img.shields.io/badge/%20imports-isort-%231674b1?style=flat&labelColor=ef8336)](https://pycqa.github.io/isort/)
 [![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
 
 ```
 TITLE CARD:
-                               "ScreenPy Selenium"
+                              "ScreenPy Selenium"                               
```

Comment:
Such a minor thing, but this trailing empty space is gonna be removed by my text editor on my next commit i promise you.